### PR TITLE
Add missing plural inflections

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
I've left out `webaservers->webservers, web servers,` because I cannot find any occurrence out there (except a `webaservers.net` domain) and this typo seems highly unlikely to me.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -32119,9 +32119,11 @@ wayoint->waypoint
 wayoints->waypoints
 wayword->wayward
 weant->want, wean,
+weaponaries->weaponries
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:32122]: I like @vikivivi solution as long as weaponries is a valid correction.

File: codespell_lib/data/dictionary_rare.txt [hunk 2]
```diff
@@ -2,6 +2,7 @@ afterwords->afterwards
 amination->animation, lamination,
 aminations->animations, laminations,
 aminator->animator, laminator,
+aminators->animators, laminators,
```
Comment (peternewman) [inline codespell_lib/data/dictionary_rare.txt:5]: @luzpaz are these both valid, I could only find the others.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt]: I've chanegd this to `weaponaries->weaponry, weaponries,`

File: codespell_lib/data/dictionary_rare.txt (hunk 2 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary_rare.txt:5]: Are you asking whether _animators_ and _laminators_ are valid? They are just the plural inflections of _[animator](https://www.collinsdictionary.com/dictionary/english/animator)_ and _[laminator](https://www.collinsdictionary.com/dictionary/english/laminate)_, I don't see how they cannot be valid.

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary_rare.txt:5]: That said, it is true that _laminator_ is missing from the [Cambridge](https://www.google.com/search?q=site%3Adictionary.cambridge.org+laminator), [Oxford](https://www.google.com/search?q=site%3Aoxfordlearnersdictionaries.com+laminator), [MacMillan](https://www.google.com/search?q=site%3Amacmillandictionary.com+laminator) or [Longman](https://www.google.com/search?q=site%3Aldoceonline.com+laminator) online dictionaries, 
but not from the [Collins](https://www.collinsdictionary.com/dictionary/english/laminate), [Merriam-Webster](https://www.merriam-webster.com/dictionary/laminate) or [American Heritage](https://www.ahdictionary.com/word/search.html?q=laminator) dictionaries. The plural form is implicit, if you require explicit mentions of it, you will find occurrences in [Wikipedia](https://en.wikipedia.org/wiki/Lamination), [Vocabulary.com](https://www.vocabulary.com/dictionary/laminator), [The Free Dictionary](https://www.thefreedictionary.com/laminators) or [YourDictionary](https://www.yourdictionary.com/laminators).

**Comment by skangas** [comment]
LGTM.

**Comment by DimitriPapadopoulos** [comment]
Rebased.

**Comment by DimitriPapadopoulos** [comment]
Rebased.

**Comment by DimitriPapadopoulos** [comment]
@luzpaz Time permitting, could you have a look at this merge request?

**Pull request closed without merging**